<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapWidget</class>
 <widget class="QWidget" name="MapWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="imageLB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSlider" name="layerSlider">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>10</y>
     <width>30</width>
     <height>768</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>790</y>
     <width>1100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
